elasticdl/python/worker/worker.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         model_params="",
         prediction_outputs_processor="PredictionOutputsProcessor",
         max_minibatch_retry_num=DEFAULT_MAX_MINIBATCH_RETRY_NUM,
+        get_model_frequency=1,
```

Comment:
Also need to assign `get_model_frequency` in __init__ to self.

Reply:
I planed to do this in upcoming commits.